scripts.js

```diff
@@ -0,0 +1,145 @@
+const $ = window.jQuery;
+
+const gameBoard = (() => {
+  const board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' '];
+
+  const getElement = (index) => board[index];
+
+  const setElement = (data, index) => {
+    board[index] = data;
+  };
+
+  const resetBoard = () => {
+    board.forEach((element, index) => {
+      setElement(' ', index);
+    });
+  };
+
+  return {
+    getElement,
+    setElement,
+    resetBoard,
+    board,
+  };
+})();
+
+const Player = (name1, symbol1) => {
+  let name = name1;
+  const symbol = symbol1;
+  const getName = () => name;
+  const getSymbol = () => symbol;
+  const setName = (playerName) => { name = playerName; };
+  return {
+    getName,
+    getSymbol,
+    setName,
+  };
+};
+
+const displayController = (() => {
+  const updateGameInfo = (name) => {
+    const gameInfo = document.querySelector('.game-info');
+    gameInfo.classList = 'game-info';
+    document.querySelector('.game-info .current-player').textContent = name;
+  };
+
+  const renderBoard = (name) => {
+    const gridContainer = document.querySelector('.grid-container');
+    gridContainer.classList = 'grid-container';
+    updateGameInfo(name);
+  };
+
+  const displayModalMessage = (title, message) => {
+    document.querySelector('#messageModalLabel').innerText = title;
+    document.querySelector('#modalBodyMessage').innerText = message;
+    $('#messageModal').modal();
+  };
+
+  return {
+    displayModalMessage,
+    renderBoard,
+    updateGameInfo,
+  };
+})();
```

Comment:
**Note To Improve!**

Let's separate the DOM from application logic, you can create another file for DOM.

![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623090/